Services/CO.CDP.EntityVerification/Persistence/EntityVerificationContext.cs

```diff
@@ -7,9 +8,11 @@
 
 namespace CO.CDP.EntityVerification.Persistence;
 
+
 public class EntityVerificationContext : DbContext
 {
     public DbSet<Ppon> Ppons { get; set; } = null!;
+    public DbSet<Identifier> Identifiers { get; set; } = null!;
```

Comment:
If we never query Identifiers directly, do we need a DbSet for them here? Forgive me my limitted EF Core skills.